test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -211,15 +218,17 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 					Enabled: pointer.Bool(false),
 				}
 
-				By("Updating the performance profile")
-				profiles.UpdateWithRetry(profile)
-
-				testlog.Infof("Applying changes in performance profile and waiting until %s will start updating", resourcePool)
-				profilesupdate.WaitForTuningUpdating(ctx, profile)
+				// If current workload hints already contains the changes skip updating
+				if !(cmp.Equal(currentWorkloadHints, profile.Spec.WorkloadHints)) {
+					By("Updating the performance profile")
+					profiles.UpdateWithRetry(profile)
 
-				testlog.Infof("Waiting when %s finishes updates", resourcePool)
-				profilesupdate.WaitForTuningUpdated(ctx, profile)
+					testlog.Infof("Applying changes in performance profile and waiting until %s will start updating", resourcePool)
+					profilesupdate.WaitForTuningUpdating(ctx, profile)
 
+					testlog.Infof("Waiting when %s finishes updates", resourcePool)
+					profilesupdate.WaitForTuningUpdated(ctx, profile)
```

Comment:
please replace with profilesupdate.PostUpdateSync

Reply:
fixed in the latest commit